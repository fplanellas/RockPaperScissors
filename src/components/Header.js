import React from 'react';
import styled from 'styled-components';
import Score from './Score';

const HeaderStyled = styled.div`
    border: 2px solid rgba(255, 255, 255, .29);
    border-radius: 10px;
    padding: 0.5em 1em;
    position:relative;
    h1{
        font-size: 20px;
        margin-top: 6px;
    }
`

const Header = () => {

  return (         
      <HeaderStyled>
          <div>
              <h1>
                  Rock Paper Scissors    
              </h1>

              <Score />

          </div>                           
      </HeaderStyled>    
  )
}

export default Header

