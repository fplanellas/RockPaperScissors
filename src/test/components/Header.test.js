import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('testing in component Header', () => {

    test('should be displayed correctly', () => {
        const wrapper = shallow( <Header /> );
        expect( wrapper ).toMatchSnapshot();
    })
      
})