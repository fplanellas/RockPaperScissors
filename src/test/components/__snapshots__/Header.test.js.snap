// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing in component Header should be displayed correctly 1`] = `
<styled.div>
  <div>
    <h1>
      Rock Paper Scissors
    </h1>
    <Score />
  </div>
</styled.div>
`;
